src/rgw/driver/rados/rgw_user.h

```diff
@@ -19,11 +19,11 @@
 
 #define RGW_USER_ANON_ID "anonymous"
 
-#define SECRET_KEY_LEN 40
-#define PUBLIC_ID_LEN 20
-#define RAND_SUBUSER_LEN 5
+constexpr auto SECRET_KEY_LEN=40;
```

Comment:
As a bit of encouraging trivia: moving this to constexpr revealed conflicting definitions elsewhere!